combined_robot_hw_tests/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(combined_robot_hw_tests)
```

Comment:
I am still wavering on this, but I see your point about hiding the test plugins and agree with your choice